lib/definitions/acceleration.js

```diff
@@ -11,20 +11,41 @@ metric = {
   }
   , 'm/s2': {
     name: {
-      singular: 'Metre per second squared'
-      , plural: 'Metres per second squared'
+      singular: 'Meter per second squared'
```

Comment:
I feel like we should either write things out or use abbreviations, but not both